test/images/mnist-tpu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
```

Comment:
I don't see any content in this except the baseimage name reference here! instead, can you put a source of `gcr.io/kubernetes-e2e-test-images/mnist-tpu:1.0` here so that anyone can build and push into test registry here?

btw: where is the source for this image `gcr.io/kubernetes-e2e-test-images/mnist-tpu:1.0`?

Reply:
Just amd64 is not enough, the idea of having this is to make sure we have images for all the supported architecture, list -  `amd64, arm, arm64, ppc64le`

Reply:
@mkumatag Thank you for your comments! Given this feature is moved to the next milestone, we could build a LargeImage for this purpose. This patch just reuses what's in the e2e registry.